src/navigators/TabNavigator.tsx

```diff
@@ -0,0 +1,88 @@
+import { StyleSheet } from 'react-native';
+
+import { BlurView } from '@react-native-community/blur';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+
+import { CustomIcon } from 'components/customIcon/CustomIcon';
+import { BasketScreen } from 'screens/BasketScreen';
+import { HomeScreen } from 'screens/HomeScreen';
+import { ProfileScreen } from 'screens/ProfileScreen';
+import { Colors } from 'utils/constants/Colors';
+
+const Tab = createBottomTabNavigator();
+
+export const TabNavigator = () => {
+    return (
+        <Tab.Navigator
+            initialRouteName="Home"
+            screenOptions={{
+                tabBarHideOnKeyboard: true,
+                headerShown: false,
+                tabBarShowLabel: false,
+                tabBarStyle: styles.containerTabBar,
+                tabBarBackground: () => (
+                    <BlurView overlayColor="" blurAmount={5} style={styles.blurStyles} />
+                ),
+            }}
+        >
+            <Tab.Screen
+                name="Home"
+                component={HomeScreen}
+                options={{
+                    tabBarIcon: ({ focused, color, size }) => (
+                        <CustomIcon
+                            name="home"
+                            size={25}
+                            color={focused ? Colors.gold : Colors.grey}
```

Comment:
move this logic to function and use everywhere

Reply:
fixed